charts/fluid-dataloader/alluxio/templates/configmap.yaml

```diff
@@ -58,10 +58,9 @@ data:
         checkPathExistence "$path"
         alluxio fs setReplication --max $replica -R $path
         if [[ $needLoadMetadata == 'true' ]]; then
-            time alluxio fs distributedLoad -Dalluxio.user.file.metadata.sync.interval=0 --replication $replica $path
-        else
-            time alluxio fs distributedLoad --replication $replica $path
+            time alluxio fs ls -R -Dalluxio.user.file.metadata.sync.interval=0 $path
```

Comment:
If there's large amount of files, will `alluxio fs ls -R` make memory pressure on this Pod?

Reply:
Could dataload set cpu, memory resources?